app/models/inventory.rb

```diff
@@ -6,13 +6,38 @@
 #  name        :string           not null
 #  deadline    :datetime         not null
 #  district_id :integer          not null
+#  owner_id    :integer
 #
 
 class Inventory < ActiveRecord::Base
+  include Authority::Abilities
   has_many :product_entries
   has_many :data_entries
+  has_many :access_requests, class_name: 'InventoryAccessRequest'
   belongs_to :district
+  belongs_to :owner, class_name: 'User'
+  self.authorizer_name = 'InventoryAuthorizer'
 
   accepts_nested_attributes_for :product_entries
   accepts_nested_attributes_for :data_entries
+
+  has_many :members, class_name:'InventoryMember'
+  has_many :participants, -> { where(role: 'participant') }, class_name:'InventoryMember'
+  has_many :facilitators, -> { where(role: 'facilitator') }, class_name:'InventoryMember'
```

Comment:
I think we had a conversation about this before - the user's role is apparently being mutated globally and not per assessment (that is to say, a user cannot hold two distinct roles in two distinct assessments).  This seems like a clean approach to get us away from that.